packages/web/src/pages/sign-up-page/components/MobileNavHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  Box,
+  Flex,
+  IconAudiusLogoHorizontal,
+  IconCaretLeft,
+  PlainButton,
+  iconSizes
+} from '@audius/harmony'
+import { useHistory } from 'react-router-dom'
+
+type MobileNavHeaderProps = {
+  isBackAllowed?: boolean
+}
+export const MobileNavHeader = (props: MobileNavHeaderProps) => {
+  const { isBackAllowed = true } = props
+  const history = useHistory()
+  const handleBackClick = () => {
```

Comment:
handleBack = history.goBack, so maybe whenever we call handleBackClick, we just do `onClick={history.goBack}`

Reply:
clean, I like it